### DISCUSSION TITLE
How to publish website again after clicking "unpublish" in Settings?

### DISCUSSION THREAD

**Discussion by Pandaklez** [community/community#27920] [Actions]
Hi, I'm new to using Jekyll. I wanted to publish website from github actions with Jekyll workflow, but I did not do it correctly on the first try, the action did not come through and website stopped updating when I switched back to "Deploy from a branch". So I decided to click "Unpublish" website in the settings and set up everything correctly from scratch in the same repository.

Could you explain how can I "Deploy from a branch" again after these actions in my repo? How can I undo clicking on "unpublish" button?

**Comment by jfullstackdev** [comment]
> Could you explain how can I "Deploy from a branch" again after these actions in my repo? How can I undo clicking on "unpublish" button?

you mean manually ? or through Actions ? 

you can look at the Repo settings, there is the Page there

**Comment by NKkrisz** [comment]
For anyone finding this in the future:

1. Click "Actions" tab in your repository
2. On the left there should be a "Deploy Jekyll with GitHub Pages dependencies preinstalled" or something similar, click it
3. In the middle you can see previous workflows and a button to run the workflow again, click that and it should work again!

![image](https://github.com/user-attachments/assets/262688f3-9173-4f0c-b395-2705b4263fe2)
